### DISCUSSION TITLE
Getting product dimensions adding shipping webhook

### DISCUSSION THREAD

**Discussion by tralawar** [swellstores/community#209] [Help] [answered]
I only see the shipping weight returned. Is there a way to get the dimensions? It would be nice to have dimensions on variations as well.

**Comment by sags95** [comment, marked as answer]
@tralawar shipping dimensions are not currently passed with the order webhook. So if you would want to receive these details you would have to query for these products/fields in your webhook handler. I'll pass this feedback on to the team nonetheless as an improvement.

**Reply by tralawar** [reply]
I have been using Cloudflare workers which doesn't support node-swell. I haven't been able to retrieve the shipping dimensions from the BE API using the documentation. I have been able to call `products/{id]` but it doesn't return the shipping dimensions like it does on `admin/console?get/products`

**Reply by sags95** [reply]
@tralawar Is it possible these are not set on all products? I've queried a few products from your store using the Backend API and I was able to retrieve the `shipment_dimensions` field in the response.

**Reply by tralawar** [reply]
Thank you for the response. I wasn't able to use swell-node. I did however figure out that the URL structure was different than the other calls I have been making 

BE API:
`https://${storeId}:${secretKey}@api.swell.store/products/${productId}`

FE API:
`https://${storeId}.swell.store/api/products/${productId}`

Using these options for both I got different responses:
```
     const requestOptions = {
       method: 'GET',
       headers: {
         'Content-Type': 'application/json',
         'Authorization': `Basic ${btoa(storeId + ':' + secretKey)}`,
             },
     };
```

**Reply by sags95** [reply]
@tralawar Ah yes it's normal that the FE and BE API return different fields
